### PR TITLE
add proxy tests

### PR DESCRIPTION
**Pull request by slangrock**

### CODE REVIEW

**Comment by mjmottram** [comment]
Er, sorry for not getting around to look at this!  I completely forgot.

File: proxy_test/jobIDfile_gl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+###Submitted Job Ids###
```
Comment (mjmottram) [inline proxy_test/jobIDfile_gl]: Did you accidentally add all files?  The job ID files should not be added (they're unique to each time you submit jobs).

**Comment by mjmottram** [comment]
All looks fine aside from the JID files.

**Comment by slangrock** [comment]
Removed the JobID files.

**Pull request merged**